### ISSUE TITLE
ISO Week Bug in January 1993

### ISSUE THREAD

**Issue by EdGaere** [python-babel/babel#806]
# Overview Description
There appears to be a bug when generating the ISO year-week for the first week of Jan 1993 for locale en_US.

8th Jan 1993 is the first ISO week of 1993, but Babel produces _1993w2_ for format _"Y'w'w"_ and locale _en_US_
```
cal 1993-01-08
```

```
     January
Su Mo Tu We Th Fr Sa
                1  2
 3  4  5  6  7  8  9
```

# Steps to Reproduce
```
from datetime import date
from babel.dates import format_date, format_datetime, format_time

d = date(1993, 1, 8)

for locale_name in ["en_US", "en_GB", "de_DE", "fr_CH", "sv_SE", "es_ES", "nn_NO", "it_IT", "nl_NL", "pt_PT"]:
    s = format_date(d, locale=locale_name, format="Y'w'w") # ISO
    print(f"{d} | {locale_name} | {s}")
```

# Actual Results
1993-01-08 | en_US | 1993w2 <-bug
1993-01-08 | en_GB | 1993w1
1993-01-08 | de_DE | 1993w1
...

# Expected Results
1993-01-08 | en_US | 1993w1  <-expected
1993-01-08 | en_GB | 1993w1
1993-01-08 | de_DE | 1993w1
...

I would have expected all locales to have the same ISO Year Week since ISO is intended to be universal, no ?

# Reproducibility

# Additional Information:

**Comment by tatarize** [comment]
No. ISO week is intended to reflect cultural understanding as best as they can.

This is not a bug. Day 1 is absolutely in Week 1. If that's a Friday, well then that's your week. 1 day in a week counts as a week. So the 2nd Friday in the month is absolutely in Week 2.

This relates to the locale for `self.locale.min_week_days` in en_US is 1. Most other places have 4.

**Comment by tatarize** [comment]
In the US look at a calendar, anything in that first row is week 1. Any days in the second row is week 2.
